src/index.html

```diff
@@ -15,8 +15,101 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
```

Comment:
Ensure to add empty lines between different HTML blocks to improve readability. For example, include an empty line between the header and nav elements, as well as between nav and main, and so on.

for example
```
<header>
  <!-- Content of header -->
</header>

<nav>
  <!-- Content of navigation -->
</nav>
```